### PR TITLE
Sk/feat/events landing page

### PR DESCRIPTION
**Pull request by shawshankkumar**
image

image

image

image

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -20,8 +18,7 @@ const AboutUs = ({ timelineData }) => {
         />
         <link rel="icon preload canonical" href="./images/kzillalogo.png" />
       </Head>
-      <Navbar />
-
+      \
```
Comment (agarwalviraj) [inline pages/about.tsx:21]: Remove this

File: shared/components/events/featured-events.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <div className="bg-black sm:pt-26 pt-24">
+      <Fade up>
+        <h1 className="text-white font-medium text-center text-4xl">
+          Featured Events
+        </h1>
+      </Fade>
+
+      <div className="flex flex-wrap items-center justify-center sm:mt-20 mt-14 pb-10">
+        {cardContent &&
+          cardContent.map((card: Card, index) => (
```
Comment (agarwalviraj) [inline shared/components/events/featured-events.tsx:26]: `cardContent.map((card, index) => (` should do the job here

File: shared/components/events/whats-new.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import Fade from 'react-reveal/Fade'
+
+import Card from '@shared/components/events/card'
+
+const WhatsNew = ({ cardContent }) => (
```
Comment (agarwalviraj) [inline shared/components/events/whats-new.tsx:5]: Add type here as well, you can create a common file containing all types and import wherever required

**Comment by shawshankkumar** [comment]
@pratham-bhatnagar @agarwalviraj amazing work people. Let's merge this!

**Comment by harshgoel05** [comment]
Finally Dec 14 comments got resolved :)

**Pull request merged**